client/src/totalRatings.jsx

```diff
@@ -6,14 +6,16 @@ import { faStar } from '@fortawesome/free-solid-svg-icons';
 const TotalRatings = ({ total, average }) => {
   const stars = [];
   for (let index = 1; index <= 5; index += 1) {
-    const color = index <= average ? 'gold' : 'grey';
-    stars[index] = <FontAwesomeIcon key={`star-${index}`} icon={faStar} color={color} />;
+    const color = index <= average ? '#FDC038' : '#A5A8AB';
+    stars[index] = (
+      <FontAwesomeIcon className="star" key={`star-${index}`} icon={faStar} color={color} />
```

Comment:
Does the star icon show up for your review rating? Mine does not (using Semantic framework), will need to go back to it later.